stream_alert/rule_processor/classifier.py

```diff
@@ -17,6 +17,7 @@
 
 from stream_alert.rule_processor import LOGGER
 from stream_alert.rule_processor.parsers import get_parser
+from stream_alert.shared.stats import timeme
```

Comment:
I keep reading this as `ti_meme`. As silly as that is, I would recommend renaming to `time_me`to be consistent